hack/verify-no-vendor-cycles.sh

```diff
@@ -18,13 +18,22 @@ set -o errexit
 set -o nounset
 set -o pipefail
 
+staging_repos=($(find staging/src/k8s.io/ -maxdepth 1 -mindepth 1 -printf "%f\n"))
```

Comment:
This doesn't work on Darwin. Would something like [this](https://github.com/kubernetes/kubernetes/blob/master/hack/update-staging-godeps.sh#L81) work instead:
```
$(ls ${KUBE_ROOT}/staging/src/k8s.io)
```